libs/src/services/creatorNode/index.js

```diff
@@ -1,9 +1,12 @@
 const axios = require('axios')
 const FormData = require('form-data')
+const tus = require('tus-js-client')
+
 const { wait } = require('../../utils')
 const uuid = require('../../utils/uuid')
 const SchemaValidator = require('../schemaValidator')
 
+const CHUNK_SIZE = 2000000 // 2MB
```

Comment:
@dmanjunath @raymondjacobson this is the chunking size I decided to use. am open to changing this if either of you think there is a better chunking size. for large tracks, this chunking size might be too small

Reply:
i think this seems reasonable. definitely shouldn't be less than 1MB or more than 2 or 3 so i think this feels right to begin with

Reply:
am interested in having this in remove config at some point